docs/fsharp/index.md

```diff
@@ -14,23 +14,21 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
 
 # F# Guide
 
-F# is a cross-platform, open source programming language for .NET which provides first-class support for functional programming, along with support of object-oriented and imperative programming.  The Visual F# compiler and tooling are Microsoft's implementation and tooling for the F# programming language, making F# a first-class member of .NET.
+F# is a functional programming language which runs on .NET.  In addition to supporting functional programming constructs, it also has object programming capabilities.  This hybrid of functional programming with object-oriented capabilities makes F# a pragmatic language for accomplishing any task.
 
 ## If You're New to F# #
 
-If you're new to F#, begin with the [Tour of F#](tour.md) to get an overview of the language.
-
-It's also recommended that you go through the [Functions as First-Class Values](introduction-to-functional-programming/functions-as-first-class-values.md)<!--[Introduction to Functional Progamming](introduction-to-functional-programming/index.md)--> to learn Functional Programming concepts which are essential to working with F#.
-
-The [Tutorials](tutorials/getting-started/index.md) also have step-by-step guides for various skill levels and features of the language.
+If you're new to F#, begin with the [Tour of F#](tour.md) to get an overview of the language and some of its programming concepts.  If you're using Visual Studio, the Tutorial project template contains the same content.
 
 ## If You're Experienced with F# #
 
-If you know your way around F#, you'll find a lot of use in the [Language Reference](language-reference/index.md), which documents each aspect of the language thoroughly, supplemented by numerous code samples.  You'll also find a lot of use in the [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference).  The F# Core Library Reference will eventually move away from MSDN and into these current docs.
+If you know your way around F#, or want to learn more about a specific language construct, you'll find a lot of use in the [Language Reference](language-reference/index.md).  It is a thorough guide of all F# language capabilities.
+
+You'll also find a lot of use in the [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference) when programming in F#.
```

Comment:
This sentence is also super informal. How about telling what they'll find in the reference?

Something like: When programming in F#, the [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference)  is a great resource for ...